src/components/SideBar.tsx

```diff
@@ -91,10 +96,16 @@ class CursorRender extends React.Component<{}, IFormState> {
       <Div onMouseMove={this.handleMouseMove} onMouseOut={this.handleMouseOut} onMouseOver={this.handleMouseOver}>
           {listOfLinks}
```

Comment:
Why list of links should be hardcoded in CursorRender? It should be a separate component